cpp/translation_db_tool.cc

```diff
@@ -99,6 +116,20 @@ void GetPossibleCategories(DbConnection * const connection, std::set<std::string
 }
 
 
+bool IsValid3LetterLanguageCode(const std::string &language_code_candidate) {
+    static std::vector<std::string> VALID_LANGUAGE_CODES{
+        "deu", "eng", "fra"
+    };
+
+    for (const auto &valid_language_code : VALID_LANGUAGE_CODES) {
```

Comment:
On 12/08/16 15:20, Oliver Obenland wrote:
>
> In cpp/translation_db_tool.cc
> <https://github.com/ubtue/ub_tools/pull/483#discussion_r74588831>:
>
> > @@ -99,6 +116,20 @@ void GetPossibleCategories(DbConnection * const connection, std::set<std::string
> >  }
> >  
> >  
> > +bool IsValid3LetterLanguageCode(const std::string &language_code_candidate) {
> > +    static std::vector<std::string> VALID_LANGUAGE_CODES{
> > +        "deu", "eng", "fra"
> > +    };
> > +
> > +    for (const auto &valid_language_code : VALID_LANGUAGE_CODES) {
>
> Isn't this the same like |return
> std::find(VALID_LANGUAGE_CODES.cbegin(), VALID_LANGUAGE_CODES.cend(),
> language_code_candidate) != VALID_LANGUAGE_CODES.end()|?
>
Indeed it is.  Good catch!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ubtue/ub_tools/pull/483/files/35d3b8b00d976df175d0011e2470aa4cfb5435ee#r74588831>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA8gC9XVju4nUN_W2iawXI1VyySi1a75ks5qfHMXgaJpZM4JjBOz>.
>

-- 
Dr. Johannes Ruscheinski
Universitätsbibliothek Tübingen - IT-Abteilung -
Wilhelmstr. 32, 72074 Tübingen

Tel: +49 7071 29-72820
FAX: +49 7071 29-5069
Email: johannes.ruscheinski@uni-tuebingen.de